docs/play/cdc-pay-integration.md

```diff
@@ -0,0 +1,194 @@
+# CDC Pay Integration
+This page illustrates some integration options for the CDC Pay module. The CDC Pay module can be intergated directly into your game development environment.
+
+## Pre-requisites
+
+- Crypto.com Pay Developer [access](https://merchant.crypto.com/users/sign_in/)
+- Optional:
+  - Webhook server (e.g. nodejs, [ngrok](https://ngrok.com/)) 
+
+### Overview
+The CDC Pay module makes use of REST API calls to trigger certain actions. These action will create events that are retrievable via webhooks. The following shell command illsutrates how to initiate the CDC Pay payment.
+
+```sh
+POST https://pay.crypto.com/api/payments
+```
+```sh
+curl https://pay.crypto.com/api/payments \
+  -u SECRET_KEY: \
```

Comment:
Suggest to have `{{PUBLISHABLE_KEY}}` instead of `SECRET_KEY:` here, seems more clear.

Reply:
I took this info from the main CDC Pay documentation. Does this work with the Publishable key. If yes I can change it and perhaps suggest this change on the CDC PAY page as well

<img width="1234" alt="Screenshot 2022-07-15 at 11 37 23" src="https://user-images.githubusercontent.com/92729550/179186469-b9c42d7d-aaca-4792-a861-e8d3cdf3ac8a.png">

Reply:
https://pay-docs.crypto.com/#api-reference-resources-payments-create-a-payment
>  This API can be called using either your Secret Key or Publishable Key.

Reply:
Yes I tried It does work with both `KEY`, may be changed to sth like `{{PUBLISHABLE_KEY or SECRET_KEY}}`

Reply:
given this is most likely the untrusted client side, it should be the publishable key

Reply:
ok than this is updated